src/Model/Shopgate/Order.php

```diff
@@ -109,4 +107,33 @@ public function save()
     {
         $this->_getResource()->save($this);
     }
+
+    /**
+     * Get all shipments for the order
+     *
+     * @return array
+     */
+    public function getReportedShippingCollections()
+    {
+        $data = $this->getData('reported_shipping_collections');
+        $data = unserialize($data);
```

Comment:
you should use the helper class that we implemented to serialize/unserialize objects (json encode), think it's called encoder or something. It will encode as json which is more secure as far as mage2.2 is concerned or maybe it had something to do with performance.